src/primo/browzine-primo-adapter.js

```diff
@@ -202,20 +198,81 @@ browzine.primo = (function() {
     return directToPDFUrl;
   };
 
+  function showJournalCoverImages() {
+    var featureEnabled = false;
+    var config = browzine.journalCoverImagesEnabled;
+
+    if(typeof config === "undefined" || config === null || config === true) {
+      featureEnabled = true;
+    }
+
+    return featureEnabled;
+  };
+
+  function showJournalBrowZineWebLinkText() {
+    var featureEnabled = false;
+    var config = browzine.journalBrowZineWebLinkTextEnabled;
+
+    if(typeof config === "undefined" || config === null || config === true) {
+      featureEnabled = true;
+    }
+
+    return featureEnabled;
+  };
+
+  function showArticleBrowZineWebLinkText() {
+    var featureEnabled = false;
+    var config = browzine.articleBrowZineWebLinkTextEnabled;
+
+    if(typeof config === "undefined" || config === null || config === true) {
+      featureEnabled = true;
+    }
+
+    return featureEnabled;
+  };
+
   function showDirectToPDFLink() {
-    var enableShowDirectToPDFLink = false;
+    var featureEnabled = false;
     var config = browzine.articlePDFDownloadLinkEnabled;
     var prefixConfig = browzine.primoArticlePDFDownloadLinkEnabled;
 
     if(typeof config === "undefined" || config === null || config === true) {
-      enableShowDirectToPDFLink = true;
+      featureEnabled = true;
     }
 
     if(typeof prefixConfig !== "undefined" && prefixConfig !== null && prefixConfig === false) {
-      enableShowDirectToPDFLink = false;
+      featureEnabled = false;
+    }
+
+    return featureEnabled;
+  };
+
+  function showPrintRecords() {
+    var featureEnabled = false;
+    var config = browzine.printRecordsIntegrationEnabled;
+
+    if(typeof config === "undefined" || config === null || config === true) {
+      featureEnabled = true;
     }
 
-    return enableShowDirectToPDFLink;
+    return featureEnabled;
+  };
+
+  function isFiltered(scope) {
```

Comment:
By `isFiltered`, what are we filtering on? Looks like this is specifically for print records, but is it more than that?

Maybe a little tweak to the function name could make it a little quicker to figure that out. Perhaps `isFilteredOutDueToPrint` if it's just based on print?